src/sdlwindow.cpp

```diff
@@ -181,6 +183,18 @@ void inputSDLThreadRun( void )
 	SDL_Surface *icon_surface = nullptr;
 	SDL_Cursor *cursor = nullptr;
 
+	
+	{
+		bool sdlLatch_cached = g_sdlLatch.load(std::memory_order_relaxed);
```

Comment:
You can just make this a `wait`, right?

Also, no need for the relaxed/acquire anyway, being seq_cst is fine for startup.

Reply:
> You can just make this a `wait`, right?
> 
> Also, no need for the relaxed/acquire anyway, being seq_cst is fine for startup.


Ok I simplified it to just the `wait`